src/app/_components/OrbitAmbassadorCard.tsx

```diff
@@ -0,0 +1,60 @@
+import clsx from 'clsx'
+
+import { BasicCard } from '@/components/BasicCard'
+import { StaticImage, type StaticImageProps } from '@/components/StaticImage'
+
+export type OrbitAmbassadorCardProps = {
+  index: number
+  image: StaticImageProps
+  name: string
+  testimonial: Array<string> | Array<JSX.Element>
+  location: string
+}
+
+import { buildImageSizeProp } from '@/utils/buildImageSizeProp'
```

Comment:
this one ended up below `OrbitAmbassadorCardProps`, should be with other import statements